File: eval/fixtures/legacy-directory-backed-spring-commerce/src/main/java/com/acme/legacy/orders/OrderFulfillmentRepository.java
package com.acme.legacy.orders;

public class OrderFulfillmentRepository {
  public OrderRepriceResult repriceFulfillmentBacklog(String orderId) {
    String sql = "SELECT * FROM orders.spring_order_fulfillment_backlog WHERE order_id = ?";
    return new OrderRepriceResult(sql);
  }
}

File: eval/fixtures/legacy-directory-backed-wcf-commerce/src/Legacy/Orders/OrderFulfillmentRepository.cs
namespace Acme.Legacy.Orders;

public class OrderFulfillmentRepository
{
  public string RepriceFulfillmentBacklog(string orderId)
  {
    string sql = "SELECT * FROM orders.wcf_order_fulfillment_backlog WHERE order_id = @orderId";
    return sql;
  }
}
